CONTRIBUTING.md

```diff
@@ -104,7 +104,13 @@ scripts. This script generates a new empty tool containing only these minimal fe
                  << "3dCurveViewer -C -b 1 -3 -2 -c ${DGtal}/examples/samples/sinus.dat\n";
       return 0;
 ```
-
+  - doxygen tool documentation: you have to add the documentation in the doxygen format including the following elements:
+    - brief description of the tool,
+    - main usage,
+    - list of the options,
+    - typical example of use with result illustration (if available)
+   
+   You can use as model the documentation of one tool of the main DGtalTools [documentation](http://dgtal.org/doc/tools/nightly/) and start from the template generated by the ```newtools.sh```script.
```

Comment:
DGtalScripts is not really well exposed.. maybe nobody will understand what you mean by "newtools.sh"